content/post/kenobi-boot2-root-writeup.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kenobi Boot2Root Write-Up"
+date: 2020-11-14T13:43:28-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Kenobi host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Kenobi boot2root was a ton of fun because it required multiple pivots to learn enough to leak a key. On my first pass i ended up overlooking the NFS shares which really impressed the importance of carefully reviewing the scans.
```

Comment:
On my first pass I ended up overlooking several NFS shares which really instilled the importance of carefully reviewing scans.